sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -69,7 +69,15 @@ public LogsQueryClient(Uri endpoint, TokenCredential credential, LogsQueryClient
 
             Endpoint = endpoint;
             options ??= new LogsQueryClientOptions();
-            var scope = $"{endpoint.AbsoluteUri}/.default";
+            string scope;
+            if (string.IsNullOrEmpty(options.Audience?.ToString()))
+            {
+                scope = $"{endpoint.AbsoluteUri}/.default";
```

Comment:
Is the endpoint always "api.loganalytics.azure.com" even when targeting a non-public cloud resource?